sips/sip-config-waiting-period.md

```diff
@@ -0,0 +1,84 @@
+---
+sip: 116
+title: Conditionally skipping fee reclamation
+status: Proposed
+author:
+discussions-to:
+created: 2020-02-25
+---
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Simply describe the outcome the proposed changes intends to achieve. This should be non-technical and accessible to a casual community member.-->
+
+Allow waiting period to be set to zero and avoid unecessary fee reclamation functionality.
+
+## Abstract
+
+<!--A short (~200 word) description of the proposed change, the abstract should clearly describe the proposed change. This is what *will* be done if the SIP is implemented, not *why* it should be done or *how* it will be done. If the SIP proposes deploying a new contract, write, "we propose to deploy a new contract that will do x".-->
+
+[SIP-114](./sip-114.md) will enable synth exchanging on L2. Given that transaction (and hence oracle) frontrunning is not possible on Optimisic Ethereum (OE), there is no need for storing exchanges and settling them after a certain amount of time has elapsed. This SIP proposes to set the waiting period to 0 when applicable, as is the case with Optimism.
```

Comment:
frontrunning is much less likely on L2 but technically could still be possible.

Reply:
And setting it to 0 would technically be an SCCP - this is a SIP that contains the code changes to skip fee reclamation if and when the waiting period is 0